README.md

```diff
@@ -712,7 +713,7 @@ The following table lists the parameters for the `devicemonitor` component and t
 | `devicemonitor.automigrate` | Enable automatic database migrations at service start up | `true` |
 | `devicemonitor.image.registry` | Docker image registry | `registry.mender.io` |
 | `devicemonitor.image.repository` | Docker image repository | `mendersoftware/devicemonitor` |
-| `devicemonitor.image.tag` | Docker image tag | `mender-3.5.0` |
+| `devicemonitor.image.tag` | Docker image tag | `nil` |
```

Comment:
I am wondering why do we default to null? should there be some default? arent we defaulting to global tag?

Reply:
We could actually set mender tags by default, but would complex the setup a bit.
My proposal is just setting `mender-3.*.*` once in the global; for example: https://github.com/mendersoftware/mender-docs/pull/2102/files#diff-30ea1193d703087fda20ccd8993a3ae9bedca62db6d539e8833a7ba589026f65R62